src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -62,18 +71,38 @@ public ObjectMapper objectMapper() {
 
     @Override
     public void addInterceptors(final InterceptorRegistry registry) {
-        registry.addInterceptor(loggingInterceptor)
-                .excludePathPatterns("/healthcheck");
-
-        registry.addInterceptor(transactionInterceptor)
-                .addPathPatterns("/transactions/{transactionId}/**");
 
+        registry.addInterceptor(loggingInterceptor)
+            .excludePathPatterns("/healthcheck");
+
+        registry.addInterceptor(transactionInterceptor).order(Ordered.HIGHEST_PRECEDENCE)
```

Comment:
Ordering should be managed by the order that interceptors are registered and the logging interceptor should be left as the highest priority so it catches the entire time for the request.

Reply:
my mistake. I fixed the merged issue and I didnt fix the logic.